ios/browser/api/bookmarks/brave_bookmarks_api.mm

```diff
@@ -43,8 +45,9 @@ - (instancetype)initWithNode:(IOSBookmarkNode*)node
 @end
 
 @interface IOSBookmarkNode () {
-  const bookmarks::BookmarkNode* node_;  // UNOWNED
+  const bookmarks::BookmarkNode* node_;
```

Comment:
this is NOT OWNED

Reply:
I guess it is sometimes NOT OWNED so as discussed in DM let's do a follow-up to separate the normal bookmark code from the import/export to avoid potential issues here